### PR TITLE
Added a file for a C++ guide  

### PR DESCRIPTION
**Pull request by MrAsrani1312**
A guide for C++ has been added.
- Introduction to c++

### CODE REVIEW

File: src/pages/articles/cplusplus/Introduction to C++/index.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+* Dont forget to tell the computer that this is the last line it needs to execute. To do this , you add the closing curly brace "}" . You may encounter an error before program execution if you do not finish your program with **}** .
+
+### The code should look something like this (with some descreption):
+
+![Image of snippet ](https://github.com/MrAsrani1312/Images/blob/master/snip1.PNG)
+
+
+Programmers use a Hello World program (like this one) as a ritual on using a new programming language. It is a symbol of good luck.  
+_You have finished coding your first C++ program and have mostly understood the code you have written/typed. CONGRATULATIONS!_
+</br> **Good Luck to all of you**
```
Comment (Bouncey) [inline src/pages/articles/cplusplus/Introduction to C++/index.md]: We don't implicitly need `<br />` tags in a markdown file. Just hit return for a new line

File: src/pages/articles/cplusplus/Introduction to C++/index.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+### The code should look something like this (with some descreption):
+
+![Image of snippet ](https://github.com/MrAsrani1312/Images/blob/master/snip1.PNG)
+
+
+Programmers use a Hello World program (like this one) as a ritual on using a new programming language. It is a symbol of good luck.  
+_You have finished coding your first C++ program and have mostly understood the code you have written/typed. CONGRATULATIONS!_
+</br> **Good Luck to all of you** 
+</br> **Happy Coding ! :)**
+</br> **Feel free to ask any queries on FreeCodeCamp's GitHub page or DM me on Twitter**
```
Comment (Bouncey) [inline src/pages/articles/cplusplus/Introduction to C++/index.md]: I think we would be better directing queries towards the [freeCodeCamp forum](https://forum.freecodecamp.org)

File: src/pages/articles/cplusplus/Introduction to C++/index.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+```C++
+#include <iostream>
+using namespace std ;
+int main()
+{
+    cout<<"Hello World"<<endl;
+    return 0;
+}
+```
+
```
Comment (Bouncey) [inline src/pages/articles/cplusplus/Introduction to C++/index.md]: How about using a [repl.it](https://repl.it/) embed to use C++ in the browser with the above program and make this article more interactive?

File: src/pages/articles/cplusplus/Introduction to C++/index.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
+---
```
Comment (Bouncey) [inline src/pages/articles/cplusplus/Introduction to C++/index.md]: We require the folder structure to be in kebab-case. i.e. `introduction-to-cplusplus`

File: src/pages/articles/cplusplus/Introduction to C++/index.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+using namespace std ;
+int main()
+{
+    cout<<"Hello World"<<endl;
+    return 0;
+}
+```
+
+### The Output of this program will simply be :
+![Image of output](https://github.com/MrAsrani1312/Images/blob/master/snip1out.PNG)
```
Comment (Bouncey) [inline src/pages/articles/cplusplus/Introduction to C++/index.md]: Could this be replaced by a [repl.it](https://repl.it/) embed?

**Comment by MrAsrani1312** [comment]
Ok I'll make the required changes...it will take a few days because I'm having some network issues...

**Comment by Bouncey** [comment]
@MrAsrani1312 it looks like we cannot embed iframes at the minute, so the repl.it embeds will not work.

**Comment by MrAsrani1312** [comment]
@Bouncey  Ok...so what do I use currently? I'll message again when a couple of more doc's are done. We xan update the files later again too.
 😁

**Comment by Bouncey** [comment]
@MrAsrani1312 Sorry for the delay, I didn't realise you had updated the PR.

Thanks for this great article, I hope the delay has not put you off writing more!

**Comment by Bouncey** [comment]
## LGTM 👍 

Thanks for contribution @MrAsrani1312 ✨ ✨

**Pull request merged**